ene/files.py

```diff
@@ -17,14 +17,12 @@
 """This module handles local video files."""
 
 import re
-from collections import defaultdict
 from os import walk
 from pathlib import Path
 from typing import Iterable
+from ene.models import Show, Episode, ShowList, init_db, db
```

Comment:
global db import behavior is pretty bad, this should be injected into FileManager instead.